src/index.js

```diff
@@ -1,16 +1,26 @@
-/* eslint-disable import/no-unresolved */
 const express = require('express');
 const bodyParser = require('body-parser');
-
 const app = express();
+const controller = require('./server/controllers/controller.js')
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+
 app.get('/', (req, res) => {
-  res.json({ Hello: 'World' });
+  let input = 'brasilia'
```

Comment:
Essa função é a principal?

Reply:
Se for a função principal não é para estar no index. Deem uma olhada no que a galera de clima fez, eles não precisaram tocar no index.